resharper/src/resharper-unity/Unity3dRider/Assets/Plugins/Editor/JetBrains/RiderPlugin.cs

```diff
@@ -467,16 +480,17 @@ static void RiderPreferencesItem()
       
       EditorGUI.BeginChangeCheck();
 
-      var help = @"For now target 4.5 is strongly recommended.
+      var help = @"TargetFramework >= 4.5 is strongly recommended.
  - Without 4.5:
     - Rider will fail to resolve System.Linq on Mac/Linux
     - Rider will fail to resolve Firebase Analytics.
  - With 4.5 Rider will show ambiguous references in UniRx.
 All those problems will go away after Unity upgrades to mono4.";
-      TargetFrameworkVersion45 =
-        EditorGUILayout.Toggle(
-          new GUIContent("TargetFrameworkVersion 4.5",
-            help), TargetFrameworkVersion45);
+           
+      TargetFrameworkVersion =
```

Comment:
default here is TargetFrameworkVersion, it has default value in the getter
EditorPrefs.GetBool("Rider_TargetFrameworkVersion45", true)?"4.5":"3.5")